examples/supported-python-versions-example/multiple-python-version-usage/README.md

```diff
@@ -0,0 +1,139 @@
+# Supported Python Versions Example - Multiple Python Version Usage
+
+Overview
+--------
+
+In this part of the Supported Python Versions example, you will deploy a test
+data job, check what python version it is deployed with, and change the
+python version for its deployment.
+
+This part is a continuation of the [Control Service Setup](https://github.com/vmware/versatile-data-kit/tree/main/examples/supported-python-versions-example/control-service-setup), and assumes that the Control Service instance supports
+`3.9-pre-release` and `3.9-release`. If the Service instance that you're using does
+not support the aforementioned python versions, follow the above link to update
+the Control Service.
+
+Data Job Code
+-------------
+
+The source code of the data job used in this example can be found [here](https://github.com/vmware/versatile-data-kit/tree/main/examples/supported-python-versions-example/multiple-python-version-usage).
+
+It is a simple job with only one step that prints a log message:
+<details>
+    <summary>20_python_step.py</summary>
+
+```python
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+from vdk.api.job_input import IJobInput
+
+log = logging.getLogger(__name__)
+
+
+def run(job_input: IJobInput):
+    log.info(f"Starting job step {__name__}")
+```
+</details>
+
+<details>
+    <summary>config.ini</summary>
+
+```ini
+; Information about the owner of the Data Job
+[owner]
+
+; Team is a way to group Data Jobs that belonged to the same team.
+team = my-team
+
+; Configuration related to running data jobs
+[job]
+schedule_cron = 11 23 5 8 1
+
+; Who will be contacted and on what occasion
+[contacts]
+notified_on_job_failure_user_error=
+notified_on_job_failure_platform_error=
+notified_on_job_success=
+notified_on_job_deploy=
+```
+</details>
+
+Usage
+-----
+
+Save the above data job code into a directory named `test-job` or open a
+terminal and navigate to the directory of this part of the example. Once
+ready, create a data job like so:
+
+```shell
+vdk create -n test-job -t my-team -p . --no-template
+```
+
+and deploy the data job by executing:
+```shell
+vdk deploy -n test-job -t my-team -p test-job -r "new job"
+```
+
+You would need to wait for a few minutes for the job deploy to finish. Once
+the data job is deployed, you can check the deployment by issuing:
+```shell
+vdk deploy --show -t 'my-team' -n 'test-job'
+```
+
+The result should be something like the following
+![deploy result](images/deploy_show1.png)
+
+The `python_version` column indicates the python version that the data job
+was deployed to use. Version `3.9-pre-release` is the default version as set
+in [Control Service Setup](https://github.com/vmware/versatile-data-kit/tree/main/examples/supported-python-versions-example/control-service-setup) part of the example.
+
+Change Python Version
```

Comment:
I think we are missing a step here. 

The user should first check what python versions are supported using `vdk info` , right? Otherwise if they try to change to unsupported version , the deployment would fail (right?)

Reply:
True, forgot about that. Updated, thanks